volatility3/framework/layers/intel.py

```diff
@@ -464,6 +464,27 @@ def _translate_swap(
                     entry=excp.entry,
                     swap_offset=swap_offset,
                 )
+            # Handle Prototype PTEs
+            if isinstance(self, WindowsIntel32e):
+                if pbit:           
+                    vaddr = (0xffff << self._maxvirtaddr) | (entry >> 16)
```

Comment:
Also, are the `0xffff` in order to canonicalize the address?  There's a [separate function](https://github.com/volatilityfoundation/volatility3/blob/a4b927f478eca745a8a4c7ec25c77fa6de49d0df/volatility3/framework/layers/intel.py#L119) for doing that on intel layers...